### DISCUSSION TITLE
Enter Console on Truecharts instance

### DISCUSSION THREAD

**Discussion by abishur** [itzg/docker-minecraft-bedrock-server#272] [Q&A]
I'm using Truecharts to create an instance of minecraft bedrock on TrueNAS.  I'd like to add some people to my whitelist but that's a console only command.  I can easily get into the bash prompt of the container, but how do I send commands to the minecraft console at that point?

**Comment by itzg** [comment]
Try out this technique for now

https://github.com/itzg/docker-minecraft-bedrock-server/issues/15#issuecomment-1197622695
